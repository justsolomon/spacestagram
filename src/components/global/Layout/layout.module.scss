@import "styles/variables";

.main-content {
  margin: 1rem 3rem;

  @media #{$media-desktop-sm} {
    margin: 1rem 2.5rem;
  }

  @media #{$media-tablet} {
    margin: 1rem 2rem;
  }

  @media #{$media-mobile} {
    margin: 1rem 1.5rem;
  }
}
