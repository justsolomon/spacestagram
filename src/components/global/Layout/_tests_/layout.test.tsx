import { render } from "@testing-library/react";
import Layout from "..";

describe("<Layout />", () => {
  it("should render the main layout content correctly", () => {
    const { asFragment } = render(<Layout children="Content here" />);

    expect(asFragment).toMatchSnapshot();
  });
});
