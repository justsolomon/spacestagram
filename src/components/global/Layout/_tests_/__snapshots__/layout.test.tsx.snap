// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> should render the main layout content correctly 1`] = `[Function]`;
